fashrevwall/run_streaming.py

```diff
@@ -0,0 +1,8 @@
+from TwitterClient import TwitterClient
```

Comment:
Whatever you think is best, the client is ready for any of the options (streaming non stop or get tweets every X minutes).